types/params.go

```diff
@@ -340,6 +338,8 @@ func (params ConsensusParams) HashConsensusParams() []byte {
 		panic(err)
 	}
 
+	hasher := sha256.New()
```

Comment:
We could simplify even further:

```suggestion
	sum := sha256.Sum256(bz)
	return sum[:]
```